cmd/oras/pull.go

```diff
@@ -133,6 +135,13 @@ func runPull(opts pullOptions) error {
 				}
 				// Skip s if s is unnamed and has no successors.
 				if len(ss) == 0 {
+					if _, exists := printed.Load(s.Digest.String()); !exists {
```

Comment:
Should use https://pkg.go.dev/sync#Map.LoadOrStore instead of .Load
```suggestion
					if _, existed:= printed.LoadOrStore(s.Digest.String(), true); !existed {
```